docs/api/ref/schemas/product_ingredients.yaml

```diff
@@ -0,0 +1,153 @@
+type: object
+description: Fields about ingredients of a product
+properties:
+
+  additives_tags:
+    type: array
+    items:
+      type: string
+  allergens:
+    type: string
+    description: comma separated list of allergens
+  allergens_lc:
+    type: string
+    description: language in which `allergens` where input
+  allergens_hierarchy:
+    type: array
+    items:
+      type: string
+  allergens_tags:
+    type: array
+    items:
+      type: string
+
+  ingredients:
+    $ref: ./ingredient.yaml
+  ingredients_analysis:
+    type: object
+    properties:
+      "en:palm-oil":
+        type: array
+        items:
+          type: string
+      "en:vegan-status-unknown":
+        type: array
+        items:
+          type: string
+      "en:vegetarian-status-unknown":
+        type: array
+        items:
+          type: string
+  ingredients_analysis_tags:
+    type: array
+    items:
+      type: string
+  ingredients_from_or_that_may_be_from_palm_oil_n:
+    type: integer
+  ingredients_from_palm_oil_n:
+    type: integer
+  ingredients_from_palm_oil_tags:
+    type: array
+    items:
+      type: object
+  ingredients_hierarchy:
+    type: array
+    items:
+      type: string
+  ingredients_n:
+    type: integer
+  ingredients_n_tags:
+    type: array
+    items:
+      type: string
+  ingredients_original_tags:
+    type: array
+    items:
+      type: string
+  ingredients_percent_analysis:
+    type: integer
+  ingredients_tags:
+    type: array
+    items:
+      type: string
+  ingredients_text:
+    type: string
+    description: |
+      Raw list of ingredients. This will get automatically
+      parsed and get used to compute the Eco-Score or find allergens, etc..
+
+      You can either request it (if it exists) or send it in a specific language.
+
+      **TODO**: explain if it is the one in the main product language or in requested language
```

Comment:
It is a copy of the one in the main language.

Reply:
ok, thank you.